### ISSUE TITLE
Type to select column names from large catalogs

### ISSUE THREAD

**Issue by low-sky** [glue-viz/glue-vispy-viewers#126]
When catalogs grow to have >100 columns it becomes unwieldy to scroll through looking for a specific tag.  It might be nice to implement (a la Topcat) the ability to select columns for plotting in, e.g., the scatter plot tool by typing the beginning of their name.  This would turn drop-down menu items a combobox.

**Comment by astrofrog** [comment]
I agree this would be nice - we should make a base Qt class for combo boxes that supports this.

Note to self: http://www.qtcentre.org/threads/23143-Combobox-entries-filter-as-I-type
